descqa/StellarMassFunction.py

```diff
@@ -0,0 +1,353 @@
+from __future__ import print_function, division, unicode_literals, absolute_import
+import os
+import math
+try:
+    from itertools import zip_longest
+except ImportError:
+    from itertools import izip_longest as zip_longest
+
+import numpy as np
+from GCR import GCRQuery
+
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+from itertools import cycle
```

Comment:
move these two lines (`from itertools import cycle` and `from itertools import count`) to before import numpy as np

Reply:
Done.